src/rpc/blockchain.cpp

```diff
@@ -1389,6 +1391,7 @@ UniValue MempoolInfoToJSON(const CTxMemPool& pool)
     ret.pushKV("maxmempool", (int64_t) maxmempool);
     ret.pushKV("mempoolminfee", ValueFromAmount(std::max(pool.GetMinFee(maxmempool), ::minRelayTxFee).GetFeePerK()));
     ret.pushKV("minrelaytxfee", ValueFromAmount(::minRelayTxFee.GetFeePerK()));
+    ret.pushKV("unbroadcastsize", (int64_t)pool.GetUnbroadcastTxs().size());
```

Comment:
style-nit: instead of a "blind cast" I slightly prefer the one that enables compile time checks and fails compilation in case it should be impossible to cast without loss of precision.

```suggestion
    ret.pushKV("unbroadcastsize", int64_t{pool.GetUnbroadcastTxs().size())};
```

Reply:
That makes sense. The only issue is it wouldn't be consistent with all the other casts throughout the file. I'm not sure what's the best practice: let this one be inconsistent, change all of them in this PR, or change all of them in a followup PR?

Reply:
The guideline is that new code should follow best practices, old code can stay as is.

Reply:
it's tough to aim for consistency in bitcoin bc the project values careful review for any changes (thus refactor PRs are generally only considered if there's a feature being built on top) so generally, we try to improve over time as we touch lines of code. 

another example you'll see is mismatched naming conventions. code that's been touched recently adheres to the best practices, but code that's been around for longer still follows old conventions 

(tldr- improve what you touch over time instead of aiming for consistency)

Reply:
Makes sense, thank you!